### PR TITLE
Add permissions to allow cloning of Images

### PR DESCRIPTION
**Pull request by shwetaap**
Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

**What this PR does / why we need it**:
Adds permission to allow cloning of Images

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2 [hunk 1]
```diff
@@ -8,5 +8,5 @@ rules:
   resources: ["persistentvolumeclaims", "persistentvolumeclaims/status"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["cdi.kubevirt.io"]
-  resources: ["datavolumes/source"]
+  resources: ["datavolumes", "datavolumes/source"]
```
Comment (mhenriks) [inline roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2:11]: this role is fine.  It is the rolebinding that needs to be updated to include:
```
subjects:
- kind: Group
  name: system:serviceaccounts
  apiGroup: rbac.authorization.k8s.io
```

That in addition to https://github.com/kubevirt/kubevirt/pull/4269 should result in a working solution

Comment (mhenriks) [reply roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2:11]: You DO NOT want anyone to be able to create datavolumes in the golden namespace.

Were you testing with the most recent kubevirt master code?

Comment (omeryahud) [reply roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2:11]: @mhenriks Our CI is not testing with master kubevirt, it is testing with the latest release, do you know if the code changes you made made it to a release?

Comment (mhenriks) [reply roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2]: @omeryahud no

Comment (mhenriks) [reply roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2]: Also `oc auth can-i create datavolumes/source --as system:serviceaccount:default:default -n kubevirt-os-images`

Should be

`oc auth can-i create datavolumes --subresource source --as system:serviceaccount:default:default -n kubevirt-os-images`

Comment (shwetaap) [reply roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2]: Removed datavolumes

**Comment by omeryahud** [comment]
@shwetaap Code looks good, are there any other loose ends we need to take care of?
/lgtm

**Pull request merged**